guby/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['gubylive']
+ALLOWED_HOSTS = ['gubylive', 'localhost', '127.0.0.1']
```

Comment:
We can keep `localhost` here, it would be required since we'll be using NGINX as a proxy and we'll need django to understand `localhost` for that. 

Why do we need `127.0.0.1`?

Reply:
Just added 127.0.0.1 as an equiv to localhost.

Reply:
Let's keep localhost and remove the 127. I think that's more used than the IP itself. What do you think>